addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/model/Device.java

```diff
@@ -34,6 +34,7 @@
     @SerializedName("scheduled_events")
     private List<ScheduledEvent> scheduledEvents = new ArrayList<ScheduledEvent>();
     private transient Location location;
+    private List<Setting> settings = new ArrayList<Setting>();
```

Comment:
Consider using the diamond operator, a nice name for leaving the Generic type away in the instantiation.